Formula/python.rb

```diff
@@ -94,15 +94,21 @@ def install
     ldflags  = []
     cppflags = []
 
-    unless MacOS::CLT.installed?
+    if !MacOS::CLT.installed? || MacOS.version == :mojave
+      if !MacOS::CLT.installed?
+        sdk_path = MacOS.sdk_path
+      else
+        sdk_path = "#{MacOS::CLT::PKG_PATH}/SDKs/MacOSX10.14.sdk"
+      end
+
       # Help Python's build system (setuptools/pip) to build things on Xcode-only systems
       # The setup.py looks at "-isysroot" to get the sysroot (and not at --sysroot)
-      cflags   << "-isysroot #{MacOS.sdk_path}"
-      ldflags  << "-isysroot #{MacOS.sdk_path}"
-      cppflags << "-I#{MacOS.sdk_path}/usr/include" # find zlib
+      cflags  << "-isysroot #{sdk_path}"
+      cflags  << "-I/usr/include" # find zlib
```

Comment:
This change, and the one below, is because Python will automatically prepend the `sdk_path` to the include path when searching for headers. I think this may actually be more correct for Xcode-only configurations too.